PrebidMobile/PrebidMobileRendering/Prebid/PBMCore/PBMSafariOpener.m

```diff
@@ -34,10 +33,9 @@
 #import "PBMMacros.h"
 
 
-@interface PBMClickthroughBrowserOpener ()
+@interface PBMSafariOpener ()
```

Comment:
Let's rename to `PBMSafariVCOpener`. Because there is a difference between opening in the `SFSafariViewController` and the Safari application.